answerking_app/models/validation/validators.py

```diff
@@ -4,7 +4,7 @@
 from django.core.exceptions import ValidationError
 
 
-def validate_positive_number(value):
+def validate_positive_number(value: any) -> int:
```

Comment:
`any` is a standard library function not a type, you can get the any type in `typing.Any`